app/lib/fake/server/fake_analyzer_service.dart

```diff
@@ -57,6 +62,11 @@ class FakeAnalyzerService {
           });
           _logger.info('running on port $port');
 
+          (taskWorkerCloudCompute as FakeCloudCompute).startInstanceExecution();
+          await taskBackend.backfillTrackingState();
+          await taskBackend.start();
+          registerScopeExitCallback(taskBackend.stop);
```

Comment:
Oh, I've just found out that `services.dart` already has it on line 263. Do we still need it here?

Reply:
good catch... there is no reason not call it -- also no harm in doing it. Remove it!